packages/coinjoin/src/client/CoinjoinPrison.ts

```diff
@@ -133,4 +133,12 @@ export class CoinjoinPrison extends TypedEmitter<CoinjoinPrisonEvents> {
             this.dispatchChange();
         }
     }
+
+    disable() {
+        if (this.changeEventTimeout) {
+            clearTimeout(this.changeEventTimeout);
```

Comment:
what about trying to emit the change? or is it already too late when disabling?

Reply:
it will not be emitted anyway when CoinjoinClient is disabled (listeners are cleared)

Reply:
but! i can canche few more lines:
- emit change without the delay
- remove listeners right after that

Reply:
[4da74a8](https://github.com/trezor/trezor-suite/pull/8549/commits/4da74a8b8e1a92e02cb46e189539d95cf0018255)

Reply:
ok so i was not happy with this `setTimeout` throttling and came up with much cleaner solution: `setImmediate` :)

it doeas exacly what i need here triggering the event after next tick:
https://nodejs.dev/en/learn/understanding-setimmediate/

Reply:
Very cool, `setImmediate` really fits here 👍 

The fallback on setTimeout is just for development in browser, right?